Source/Icebreaker/Bot/IcebreakerBot.cs

```diff
@@ -226,22 +233,26 @@ private async Task HandleMessageActivityAsync(ITurnContext turnContext, Cancella
             try
             {
                 var activity = turnContext.Activity;
-                var senderAadId = activity.From.AadObjectId;
+                var senderId = activity.From.Id;
```

Comment:
Can you comment on why we are changing this to use the Bot Framework id instead of the AAD Id?

Reply:
When attempting to access a ChannelAccount's AadObjectId in MatchingService.cs, it always returns null (might be a bug with ChannelAccount). In order to reference the user's profiles during matching the Id had to be used to avoid this issue.